src/front-end/src/components/EmailVerify.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { Button } from 'antd';
+
+function resendVerifyEmail() {
+  // insert code to resend verification email
+  // or insert code to re-route to resend verification email page
+  window.alert('Verification Email Sent!');
+}
+
+// eslint-disable-next-line react/prefer-stateless-function
+class EmailVerify extends Component {
+  render() {
+    return (
+      <div className="EmailVerify">
+        <center>
+          <h1>
+            <font size="6">
+              A verification email has been sent to your email address!
+            </font>
+          </h1>
+        </center>
+        <br />
+        <center>
+          <h2>
+            <font size="5">
+              Please verify your email address in order to access:
+            </font>
+            <font size="4">
+              <ul>
+                <li>
+                  <i>Your Mentor Account</i>
+                </li>
+                <li>
+                  <i>The Mentor Tutorial</i>
+                </li>
+                <li>
+                  <i>Your Suggested Mentees</i>
+                </li>
+                <li>
+                  <i>Your Accepted Mentees</i>
+                </li>
+              </ul>
```

Comment:
I think it would look better if the bullet points from the list were removed. This can achieved by doing the following:
```suggestion
              <ul style={{ listStyleType: 'none' }}>
                <li>
                  <i>Your Mentor Account</i>
                </li>
                <li>
                  <i>The Mentor Tutorial</i>
                </li>
                <li>
                  <i>Your Suggested Mentees</i>
                </li>
                <li>
                  <i>Your Accepted Mentees</i>
                </li>
              </ul>
```

Reply:
Removed